src/asm/section.c

```diff
@@ -339,6 +339,9 @@ void out_SetLoadSection(char const *name, uint32_t type, uint32_t org,
 	if (currentLoadSection)
 		fatalerror("`LOAD` blocks cannot be nested\n");
 
+	if (sect_HasData(type))
+		error("`LOAD` blocks cannot be placed within a ROM section\n");
```

Comment:
I think the wording might be confusing: the blocks *themselves* can and should be in ROM, but the section they create can't. What do you think of "LOAD blocks cannot create a ROM section"?

Reply:
"Create" is the same word rgbasm(5) uses, so that makes sense.